drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1095 @@
+[note]: <> ("note to self and file editor, there are certain comments left throughout this file, for the sake of understanding the author's reason for using a certain method of approach, pls feel free to delete once acknowledged or reply edit with a better approach 🥂")
+
+# How to build a DAO on the Fuel Blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered quite an essential infrastructure in any blockchain.
+
+It is a Decentralized technology that blockchains incorporate into their validation system to automate governance rules.
```

Comment:
```
It is a Decentralized technology that blockchains incorporate into their validation system to automate governance rules.
```

In the context of Blockchains and Distributed Ledger Technologies, the concept of _"blockchains (...) validation system"_  might be used imprecisely.

Blockchain's ledger validation occurs at protocol level and not at application level. `Validation` is a process done by the nodes, not the smart contracts. 

DAOs are not a `decentralized technology` but `decentralized applications`, and they are used by users or organizations to automate governance execution of their projects and communities.

Reply:
I've made changes to correct the definitions